apps/integration_tests/selenium_cases.py

```diff
@@ -136,8 +140,8 @@ class Action(Enum):
 # SLSX login form
 SLSX_TXT_FLD_USERNAME = "username-textbox"
 SLSX_TXT_FLD_PASSWORD = "password-textbox"
-SLSX_TXT_FLD_USERNAME_VAL = "BBUser00000"
-SLSX_TXT_FLD_PASSWORD_VAL = "PW00000!"
+SLSX_TXT_FLD_USERNAME_VAL = "BBUser00001"
+SLSX_TXT_FLD_PASSWORD_VAL = "PW00001!"

```

Comment:
Seems like BBUser00000 had a password change again? Is that the reason for changing this?

Reply:
> Seems like BBUser00000 had a password change again? Is that the reason for changing this?

yea, it's good to avoid BBUser00000 now

Reply:
Yeah. I'd love to see the synthetic users stop changing PW's, lol. Not the first time.

Reply:
This seems to happen more often on BBUser00000 than the others, so to be clear, I'm totally happy changing this to BBUser00001, just wanted to clarify the reasoning behind the change. I think what Logan meant was that it would be nice if people stopped messing with the logins for the synthetic users, not that we need to not change the user/pw here. So it's okay either way, but feel free to set this back to BBUser00001.